### DISCUSSION TITLE
Crash after autolaunch

### DISCUSSION THREAD

**Discussion by imy5665** [iNavFlight/inav#8162] [General]
Hello dear inav group members!
Would you please explain me where i am wrong. 
The point is that after Autolaunch finishing the model behave very strange - one of aileron at the same time goes down makes the model doing a spiral. 
Model: Dart250G
FC: Matek F411WTE
Radio Link - ELRS 
inav 4.1 ; 5.0
[dart 250 v2.txt](https://github.com/iNavFlight/inav/files/8986158/dart.250.v2.txt)

please find diff and issue's video bellow

https://disk.yandex.com/i/RrNdL1yGXaJD9g

**Comment by MrD-RC** [comment]
Can't really see anything in that video, only the left elevon. 

1. Check the receiver page. Make sure that the stick's axis only moves the appropriate bar. And that stick left or down is 1000, right or up is 2000, and centred is 1500.
2. Go to the setup page and verify that the movement of the model on screen is the same as the physical model. Also, when the bottom of the wings are flat, the pitch is pretty much flat on the 3D model too.
3. Switch to manual mode and make sure that pitch up/down and roll left/right are correct.
4. Switch to angle model, recheck the stick movements are correct. Then verify the stabilisation direction is correct. 

Let us know the results of these checks.

**Reply by imy5665** [reply]
Hello!
Yes, you are right. The left elevon went down as soon as autolaunch sequence has finished. It is not normal.
Actually, I fix the problem by just re setting up whole controller step by step manually.

**Comment by breadoven** [comment]
But this is a hand held ground test from the video in the link, not an actual flight and crash. The behaviour of the control surfaces in these circumstances tells you little because the plane isn't free flying preventing the stabilisation from work correctly, i.e. the plane can't react to the control surface movements so the stabilisation drives them to random positions trying to maintain control.
